src/bunit.core/Rendering/ITestRenderer.cs

```diff
@@ -78,4 +78,15 @@ IReadOnlyList<IRenderedComponentBase<TComponent>> FindComponents<TComponent>(IRe
 	/// Disposes all components rendered by the <see cref="ITestRenderer" />.
 	/// </summary>
 	public void DisposeComponents();
+
+	/// <summary>
+	/// Disposes all components rendered by the <see cref="ITestRenderer" />.
+	/// </summary>
+	/// <remarks>
+	/// If a component (or any of the children) implements <see cref="IDisposable.Dispose"/>, awaiting this <see cref="Task"/> will wait until Dispose has finished.<br/>
+	/// In contrast to that, if a component (or any of the children) implements <see cref="IAsyncDisposable.DisposeAsync"/> <code>await DisposeComponentsAsync</code> will
+	/// does not wait until the underlying <see cref="ValueTask"/> is completed.<br/>
+	/// If you want to wait until all asynchronous disposables are completed, use the WaitFor-helpers.
```

Comment:
```suggestion
	/// If you want to wait until all asynchronous disposables are completed, use the WaitFor-helpers.
```

I dont think this is true? Using e.g. `WaitForState` wont work, since it will only reevaluate the predicate after a render cycle, and that does not happen after a component has been disposed, so the predicate would never pass. I think the only option is to use `Task.Delay` in this case, if the user cannot control the asyncness of the asyncdisposable code.

Reply:
Oh yeah. That is true. Yes `Task.Delay` or any awaitable and observable state from the outside (what ever that means for the concrete problem of the user). Will correct that.